addons/copy-reporter/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Copy reporter values",
-  "description": "Adds buttons to copy the value of reporter balloons and monitors.",
+  "description": "Adds buttons to copy the value of reporter balloons and monitors by right clicking the variable readout on the project.",
```

Comment:
```suggestion
  "description": "Adds copy value buttons to block reporter balloons and variable monitor context menus.",
```

Reply:
We often use "right click context menus" for clarity.

Reply:
Ok! :)